man/LassoDeployment.Rd

```diff
@@ -11,17 +11,16 @@ LassoDeployment(type, df, grainCol, predictedCol, impute, debug, cores, modelNam
 \arguments{
```

Comment:
Why are these Lasso files being changed in this PR? The clustering work is huge by itself. Please split into other PR.